README.md

```diff
@@ -81,6 +81,27 @@ affects only the place where the application is actually installed.
 
 Head over to [releases](https://github.com/lite-xl/lite-xl/releases) and download the version for your operating system.
 
+### Windows
+
+Lite XL comes with installers on Windows for typical installations.
+Alternatively, we provide ZIP archives that you can download and extract it anywhere and run directly.
+
+To make the installation portable, simply create a `user` folder where `lite-xl.exe` is located.
+
+### macOS
```

Comment:
Just to keep as a note: in #1606 we promised to ship a non-DMG version, so we should keep that in mind before releasing the next version.